### ISSUE TITLE
Add proxy

### ISSUE THREAD

**Issue by DzianisH** [brunocvcunha/instagram4j#203]
Add possibility to set proxy host to Instagram4j object. This proxy will be used to send any request to Instagram.

**Comment by brunocvcunha** [comment]
Thanks!

**Comment by DzianisH** [comment]
Hi All,
Proxy will be available in 1.8 version.
```java
        Instagram4j instagram = Instagram4j.builder()
                .username("your email")
                .password("your password")
                .proxy(new HttpHost("host.address.com", 8080, "http"))
                .build();

        instagram.setup();
        instagram.login();

        InstagramLikeRequest request = new InstagramLikeRequest(InstagramCodeUtil.fromCode("BfjRJp1nQug"));
        InstagramLikeResult result = instagram.sendRequest(request);
        System.out.println(result);
```
